app/stacks/uk-west/applications-service/terragrunt.hcl

```
@@ -55,6 +56,7 @@ inputs = {
   back_office_service_bus_register_nsip_subscription_topic_id   = dependency.back_office_ukw.outputs.servicebus_topic_register_nsip_subscription_id
   back_office_service_bus_nsip_project_update_topic_id          = dependency.back_office_ukw.outputs.service_bus_nsip_project_update_topic_id
   back_office_service_bus_nsip_representation_topic_id          = dependency.back_office_ukw.outputs.service_bus_nsip_representation_id
+  back_office_service_bus_nsip_exam_time_table_topic_id         = dependency.back_office_ukw.outputs.service_bus_nsip_exam_time_table_id
```

The output from the back-office stack is named `service_bus_nsip_exam_timetable_id`, so will need to be named as such here:
```suggestion
  back_office_service_bus_nsip_exam_time_table_topic_id         = dependency.back_office_ukw.outputs.service_bus_nsip_exam_timetable_id
```

(the output is defined in `app/stacks/uk-west/back-office/outputs.tf` line 129)